plugins/modules/rhsm_repo_info.py

```diff
@@ -64,15 +64,17 @@ def rhsm_repo_info(module):
             confp.read_file(open("/etc/yum.repos.d/%s" % file))
             if confp.has_section(repo):
                 items = dict(confp.items(repo))
-                rhsm_info.append({
-                    "name": repo,
-                    "base_url": items["baseurl"],
-                    "type": "yum-baseurl",
-                    "check_ssl": True if items['sslverify'] == '1' else False,
-                    "check_gpg": True if items['gpgcheck'] == '1' else False,
-                    "gpgkey_paths": items['gpgkey'],
-                    "state": 'present',
-                })
+                rhsm_info.append(
+                    {
+                        "name": repo,
+                        "base_url": items["baseurl"] if "baseurl" in items else None,
```

Comment:
What happens if base_url = None? Wouldn't that cause the image build to fail?

Reply:
good idea! added specific error messages